package.json

```diff
@@ -17,6 +17,7 @@
     "postcss": "8.4.28",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-query": "^3.39.3",
```

Comment:
why should we use this state manager?

Reply:
Because according to my research around this topic, "react-query" is the best for our purpose.